src/napari_matplotlib/base.py

```diff
@@ -288,7 +296,8 @@ def clear(self) -> None:
         """
         Clear the axes.
         """
-        self.axes.clear()
+        with mplstyle.context(self.mpl_style_sheet_path):
```

Comment:
Ah! TIL. 
I had some weirdness when I was clearing but I didn't have the context manager!